x/common/asset/pair.go

```diff
@@ -149,49 +149,19 @@ func (pairKeyEncoder) Decode(b []byte) (int, Pair) {
 	return i, MustNewPair(s)
 }
 
-//-----------------------------------------------------------------------------
```

Comment:
Searched LSP and and found that this type alias was largely unused, so I removed some of the dead code.